### PR TITLE
Add new video direct download field

### PR DESCRIPTION
**Pull request by coppolaemilio**
I wanted to add a new field to request people to provide a direct download link for the showreel videos. Last time I had to go one by one asking people to provide a proper high quality version of the videos so I didn't have to download directly from YouTube (pretty sure downloading from YouTube breaks ToS). 

I'm not sure if these are the required steps to add this new field, so I create this PR so you can review @winston-yallow

Alternatively, it would be nice to just have a regular html (or template file) for the front-end where I could work on the form manually instead of it being generated by django. If that's possible I would prefer that.

### CODE REVIEW

File: vote/migrations/0001_initial.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ class Migration(migrations.Migration):
                 ('game', models.CharField(default='', max_length=200)),
                 ('author_name', models.CharField(default='', max_length=200)),
                 ('video_link', models.CharField(max_length=200, unique=True, validators=[django.core.validators.URLValidator()])),
-                ('follow_me_link', models.CharField(blank=True, default='', max_length=200, validators=[django.core.validators.URLValidator()])),
+                ('video_download_link', models.CharField(max_length=200, unique=True, validators=[django.core.validators.URLValidator()])),
+                ('follow_me_link', models.CharField(blank=True, default='', max_length=200)),
```
Comment (winston-yallow) [inline vote/migrations/0001_initial.py:54]: Django migrations should not be edited by hand. Instead a new migration 0003 should be generated by running `django-admin makemigrations` to ensure database consistency. Our deployment will then be able to automatically add the column to the database. More information: https://docs.djangoproject.com/en/4.2/topics/migrations/

File: vote/models.py [hunk 2]
```diff
@@ -97,7 +97,8 @@ class Video(models.Model):
     game = models.CharField(max_length=200, blank=False, default="")
     author_name = models.CharField(max_length=200, blank=False, default="")
     video_link = models.CharField(max_length=200, blank=False, unique=True, validators=[URLValidator()])
-    follow_me_link = models.CharField(max_length=200, default="", blank=True, validators=[URLValidator()])
+    video_download_link = models.CharField(max_length=200, blank=False, unique=True, validators=[URLValidator()])
+    follow_me_link = models.CharField(max_length=200, default="", blank=True)
```
Comment (winston-yallow) [inline vote/models.py:101]: Is it intended that the URL validator is removed for the follow me link? The PR does not say anything about it

**Review by winston-yallow — COMMENTED** [review]
I've only had a quick view, I'll run this in my local test environment later

File: vote/models.py (hunk 2 above)
Comment (coppolaemilio) [reply vote/models.py:101]: yes, forgot to mention it! I believe we discussed removing the validator there, but can be re-added and maybe done in a separate PR.

Comment (winston-yallow) [reply vote/models.py:101]: no it's fine to remove it here, just wanted to make sure it's intentional

**Pull request merged**